go/arrow/cdata/cdata.go

```diff
@@ -513,14 +551,13 @@ func (imp *cimporter) importFixedSizeBuffer(bufferID int, byteWidth int64) *memo
 	return imp.importBuffer(bufferID, bufsize)
 }
 
-func (imp *cimporter) importOffsetsBuffer(bufferID int) *memory.Buffer {
-	const offsetsize = int64(arrow.Int32SizeBytes) // go doesn't implement int64 offsets yet
+func (imp *cimporter) importOffsetsBuffer(bufferID int, offsetsize int64) *memory.Buffer {
 	bufsize := offsetsize * int64((imp.arr.length + imp.arr.offset + 1))
 	return imp.importBuffer(bufferID, bufsize)
 }
 
-func (imp *cimporter) importVariableValuesBuffer(bufferID int, byteWidth int, offsets []int32) *memory.Buffer {
-	bufsize := byteWidth * int(offsets[imp.arr.length])
+func (imp *cimporter) importVariableValuesBuffer(bufferID int, byteWidth int, bytelen int) *memory.Buffer {
+	bufsize := byteWidth * bytelen
```

Comment:
Two things: 1) isn't `bytelen` a bit misleading as a name? 2) shouldn't it be an `int64`?

Reply:
fair point, i'll fix that